src/shared/components/MultiFileSelect/MultiFileSelect.tsx

```diff
@@ -222,13 +221,29 @@ export const MultiFileSelect: React.FC<MultiFileSelectProps> = ({
           variant="file"
           stepPlaceholder={files.thumbnail?.url ? <img src={files.thumbnail?.url} alt="thumbnail" /> : undefined}
           number={2}
-          title={files.thumbnail ? 'Thumbnail image' : 'Add thumbnail image'}
-          active={step === 'image'}
+          title={
+            files.thumbnail
+              ? files.thumbnail.originalBlob
+                ? (files.thumbnail.originalBlob as File).name
+                : 'Thumbnail image'
+              : 'Add thumbnail image'
+          }
```

Comment:
```suggestion
          title={
            files.thumbnail?.originalBlob
              ? (files.thumbnail.originalBlob as File).name || 'Thumbnail image'
              : 'Add thumbnail image'
          }
```
Maybe I'm missing something, but I think we could do the same as in the video, right?

Reply:
Ok, nevermind, I think I know why you did that. Let's just keep this as it is.